Packet++/src/TpktLayer.cpp

```diff
@@ -0,0 +1,54 @@
+#include "TpktLayer.h"
+//#include "CotpLayer.h"
+#include "EndianPortable.h"
+#include "TcpLayer.h"
+#include <PayloadLayer.h>
+#include <iostream>
+#include <sstream>
+#include <string.h>
+
+namespace pcpp
+{
+
+	uint8_t TpktLayer::getReserved() const { return getTpktHeader()->reserved; }
+
+	uint8_t TpktLayer::getVrsn() const { return getTpktHeader()->vrsn; }
+
+	uint16_t TpktLayer::getLength() const { return htobe16(getTpktHeader()->length); }
+
+	std::string TpktLayer::toString() const
+	{
+		std::ostringstream vrsnStream;
+		vrsnStream << getVrsn();
+		std::ostringstream reservedStream;
+		reservedStream << getReserved();
+		std::ostringstream lengthStream;
+		lengthStream << getLength();
+
+		return "TPKT Layer, version: " + vrsnStream.str() + ", reserved: " + reservedStream.str() +
+			   ", length: " + lengthStream.str();
```

Comment:
I'm not sure we need `reserved` to be part of the string... maybe just `version` and `length`?